site2/docs/reference-configuration-standalone.md

```diff
@@ -1404,13 +1404,13 @@ Deploy the schema compatibility checker for a specific schema type to enforce sc
 
 ### schemaCompatibilityStrategy
 
-The schema compatibility strategy at broker level, see [here](schema-evolution-compatibility.md#schema-compatibility-check-strategy) for available values.
+The schema compatibility strategy at broker level, see [here](https://pulsar.apache.org/docs/next/schema-evolution-compatibility/#schema-compatibility-check-strategy) for available values.
```

Comment:
I guess the reason for this change is `reference-configuration-standalone.md` is not an "internal" file, so the relative path becomes invalid, right?